libp2p/Capability.cpp

```diff
@@ -28,9 +28,10 @@ using namespace std;
 using namespace dev;
 using namespace dev::p2p;
 
-Capability::Capability(std::shared_ptr<Session> _s, HostCapabilityFace* _h, unsigned _idOffset): m_session(_s), m_hostCap(_h), m_idOffset(_idOffset)
+Capability::Capability(std::shared_ptr<Session> _s, HostCapabilityFace* _h, unsigned _idOffset, uint16_t _protocolID):
+	m_session(_s), m_hostCap(_h), m_idOffset(_idOffset), c_protocolID(_protocolID)
```

Comment:
Note: Eventually protocolID wil be stored by Session and/or another class similar to session; in the future this value would be queried and/or implicitly provided by the upper layer class rather than statically provided by Capability. The reason for this is that sub-protocols maybe dynamically added or removed which will change their identifiers, and in the future its likely the identifier becomes longer and/or arbitrary (rather than an integer).